setup.cfg

```
@@ -20,7 +20,7 @@ package_dir =
 packages = find:
 python_requires = >=3.7
 install_requires =
-    vitessce==3.0.7
+    vitessce==3.2.2
```

```suggestion
    vitessce==3.2.3
```

Related to the `scope_prefix` parameter mentioned below / in https://github.com/vitessce/vitessce-python/pull/325